CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+# Version 0.21 (TODO)
```

Comment:
Not new to this PR, but reviewing this I now notice the changelog doesn't adhere to https://keepachangelog.com/en/1.1.0/ even though it says it does. Specifically, the paragraph titles with the version numbers don't follow convention and aren't hyperlinked.

Reply:
I do not think that following that precisely is necessary. We may improve starting from a new release.

Reply:
I do agree its not *that* important, but it as a simple regex find/replace and its fixed now :+1: